service/error_handlers.py

```diff
@@ -39,21 +39,6 @@ def not_found(error):
     )
 
 
-@app.errorhandler(status.HTTP_405_METHOD_NOT_ALLOWED)
```

Comment:
Flask-RESTX will replace all of our error handlers, but this is the only one where the behavior changed.